packages/rolldown/src/rolldown-binding.wasi-browser.js

```diff
@@ -113,7 +113,7 @@ function __napi_rs_initialize_modules(__napiInstance) {
   __napiInstance.exports['__napi_register__BindingRenderedModule_struct_79']?.()
   __napiInstance.exports['__napi_register__AliasItem_struct_80']?.()
   __napiInstance.exports['__napi_register__BindingSourcemap_struct_81']?.()
-  __napiInstance.exports['__napi_register__BindingJSONSourcemap_struct_82']?.()
+  __napiInstance.exports['__napi_register__BindingJsonSourcemap_struct_82']?.()
```

Comment:
I have a question about wasi output. It looks like there are two similar looking code here under `packages/rolldown/src` and also at the package root `packages/rolldown`. 

When I build locally `just build` and `just build wasi`, the ones in `packages/rolldown/src` updated but `package/rolldown` hasn't changed. Could I be missing some local setup?

Reply:
I'm confused too. You could just leave it there for now.